scalability_and_performance/cnf-low-latency-tuning.adoc

```diff
@@ -20,7 +20,9 @@ include::modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc[level
 [role="_additional-resources"]
 .Additional resources
 
-* For more information about recommended firmware configuration, see xref:../scalability_and_performance/ztp_far_edge/ztp-vdu-validating-cluster-tuning.adoc#ztp-du-firmware-config-reference_vdu-config-ref[Recommended firmware configuration for vDU cluster hosts].
+* xref:../scalability_and_performance/ztp_far_edge/ztp-vdu-validating-cluster-tuning.adoc#ztp-du-firmware-config-reference_vdu-config-ref[Recommended firmware configuration for vDU cluster hosts].
+
+* xref:../nodes/scheduling/nodes-scheduler-node-selectors.adoc#nodes-scheduler-node-selectors[Placing pods on specific nodes using node selectors].
```

Comment:
Shouldn't this link be moved to an additional resources section directly underneath the module this applies to?

Reply:
Never mind, I see that the module includes all of those other sections.